docs/edge-compute/edge-agent-usage.md

```diff
@@ -0,0 +1,271 @@
+```
+
+```toml
+[logger]
+out = '/data/losant-edge-agent-log.log'
+level = 'verbose'
+
+[gateway]
+id = '<your-device-id>'
+key = '<your-access-key>'
+secret = '<your-access-secret>'
+
+[store]
+path = '/data/losant-edge-agent-store.db'
+```
+
+Notice that we are designating the logger to output to the same `/data` directory within the container that the store is writing to. This allows us to just mount a local folder on our host to the `/data` directory in the Agent container and have all persistent data written to a single place. Now we can run the Agent container without specifying environment variables, and instead simply provide the path to our `config.toml`.
+
+```console
+docker run -d --name docs-agent \
+  -v /var/lib/losant-edge-agent/data:/data \
+  -v /var/lib/losant-edge-agent/config.toml:/etc/losant/losant-edge-agent-config.toml \
+  losant/edge-agent
+```
+
+`/etc/losant/losant-edge-agent-config.toml` is the default location for the `toml` configuration file inside the container, so we mount our local configuration to that location in the container. Now that our Agent container is running, we can see the output by watching the log file we specified with `logger.out`:
+
+```console
+$ tail -f /var/lib/losant-edge-agent/data/losant-edge-agent-log.log
+0000-00-00T00:00:00.000Z [info] Agent Starting...
+0000-00-00T00:00:00.000Z [info] Connecting to: mqtts://broker.losant.com...
+0000-00-00T00:00:00.000Z [info] Webserver started on port: 8080
+0000-00-00T00:00:00.000Z [info] Workflows initialized and running...
+0000-00-00T00:00:00.000Z [info] Connected to mqtts://broker.losant.com
+```
+
+### Making Changes To The Config File
+
+Because we have designated a configuration file and configured our Agent container to read from it, making changes are easy. Let's say we decide that `verbose` logging is just too much logging and we only really care about `error` level logs. We can simply modify our `/var/lib/losant-edge-agent/config.toml` like so:
+
+```toml
+# ...
+[logger]
+out = '/data/losant-edge-agent-log.log'
+level = 'error'
+# ...
+```
+
+Then restart the Agent to have the Agent pick up our new configuration changes.
+
+```console
+docker restart docs-agent
+```
+
+### Webserver: HTTP Request Trigger / HTTP Response Node
+
+If you wish to enable the [HTTP Request trigger](/workflows/triggers/request/) for your Edge Compute Device, you can do so by binding a host port at the time of container creation. Building on the configuration we have going, we only need to add the `-p` flag and then configure the Agent in our configuration file to enable the trigger. See [exposing ports](https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p-expose) with Docker for more information. For more advanced configuration of the [HTTP Request trigger](/workflows/triggers/request/) and [HTTP Response Node](/workflows/outputs/http-response/), see the [Docker Hub Repo](https://hub.docker.com/r/losant/edge-agent/).
```

Comment:
Trigger should be capitalized
Suggested reword from 'Building on the configuration...' 
'In order to build on the configuration we have going...'